src/cmd/lium.bash

```diff
@@ -72,6 +72,10 @@ _lium_get_boards() {
   ${COMP_WORDS[0]} board list --cached 2>/dev/null | cut -f 1
 }
 
+_lium_get_branches() {
+  ${COMP_WORDS[0]} config show --key android_branches 2>/dev/null | sed -e 's/[]["]//g' | sed -e 's/,/ /g'  
+}
```

Comment:
Output of `lium config show` is below: 
```
$ ./target/debug/lium config show --key android_branches       
["aaa","bbb","ccc"]
```
so separate values with spaces with `sed` command: 
```
$ ./target/debug/lium config show --key android_branches | sed -e 's/[]["]//g' | sed -e 's/,/ /g'     
aaa bbb ccc
```

Reply:
I think you can make the --key parameter as optional positional. so that you can call:
```
lium config show android_branches
```

Reply:
Got it, I changed it to a positional option!

Reply:
If you change it as positional option, you should remove `--key` from this command.

Reply:
Thank you for letting me know, I removed it!!